wordpress-mobile/WordPress-Android
# Prepublishing Nudges : Fetch tags on first launch

Fixes #11687 

To test:

1. Create a new post.
2. Press publish.
3. Select the Tags field.
4. See the tags screen and the list should be populated.

### Review 
1. Only 1 reviewer is needed but anyone can review.
2. Ensure that #11634 is merged.
3. Set this base branch to feature/prepublishing_nudges
4. Remove `Not Ready for Merge`
5. Merge :)

PR submission checklist:
